CONTRIBUTING.md

```diff
@@ -30,18 +33,29 @@ faith and everyone is working towards a common goal.
 
 ## Getting Started
 
-Contributions are made to this repo via Discussions, Issues and Pull Requests (PRs). A few general guidelines that cover the trio:
+Contributions are made to this repo via Ideas, Discussions, Issues and Pull Requests (PRs).
 
+[Discussions > Ideas](https://github.com/dcgtc/dgrants/discussions/categories/ideas) - Propose and discuss ideas for new features or changes not currently in the dGrants design/architecture
+[Discussions > Architecture & Design](https://github.com/dcgtc/dgrants/discussions/categories/architecture-design) - Propose and discuss architecture and design for features or components in the current design that do not have clear or detailed definition
+[Issues](https://github.com/dcgtc/dgrants/issues) - Report problems with dGrants or add work to be done on a feature that is within the scope of the current design/architecture
```

Comment:
These render all on one line since Markdown ignores single line breaks
```suggestion
- [Discussions > Ideas](https://github.com/dcgtc/dgrants/discussions/categories/ideas) - Propose and discuss ideas for new features or changes not currently in the dGrants design/architecture
- [Discussions > Architecture & Design](https://github.com/dcgtc/dgrants/discussions/categories/architecture-design) - Propose and discuss architecture and design for features or components in the current design that do not have clear or detailed definition
- [Issues](https://github.com/dcgtc/dgrants/issues) - Report problems with dGrants or add work to be done on a feature that is within the scope of the current design/architecture
```